app/routes.py

```diff
@@ -1 +1,161 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
+from sqlalchemy.types import DateTime
+from sqlalchemy.sql.functions import now
+import requests, json
+import os
+# import logging
+# from slack_sdk import WebClient
+# from slack_sdk.errors import SlackApiError
+
+
+task_list_bp = Blueprint("task_list_bp", __name__, url_prefix="/tasks")
+
+def validate_model_task(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {model_id} invalid"}, 400))
+
+    task = cls.query.get(model_id)
+
+    if not task:
+        abort(make_response({"message": f"{cls.__name__} {model_id} not found"}, 404))
+
+    return task
+
+@task_list_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    is_valid_task = "title" in request_body and "description" in request_body
+    if not is_valid_task:
```

Comment:
Oh very interesting approach! Any reason you decided to make a variable first rather than combine the two lines together? I'm curious!